client/src/components/chess-game/ChessGame.js

```diff
@@ -15,14 +16,18 @@ import './ChessGame.css';
 */
 const Chess = typeof ChessJs === 'function' ? ChessJs : ChessJs.Chess;
 
+const getRandomNumber = (min, max) => Math.floor(Math.random() * (max - min + 1)) + min;
+
 const INIT_STATE = {
   board: [],
   playerColor: '',
   currentPlayer: WHITE_PLAYER,
   playerOneTime: DEFAULT_GAME_TIME_SECS,
   playerTwoTime: DEFAULT_GAME_TIME_SECS,
   isGameOver: false,
-  gameOverMessage: '',
+  playerOneRating: getRandomNumber(1400, 1500),
```

Comment:
Will this data be comming from server instead?

Reply:
Once we have a login implemented, will update this to get data from user details

Reply:
Please add a TODO there.